src/ts/player.d.ts

```diff
@@ -57,8 +57,19 @@ declare namespace bitmovin {
      *
      * @param eventType The type of event to subscribe to.
      * @param callback The event callback handler that will be called when the event fires.
+     *
+     * @deprecated since 7.7, use {@link PlayerAPI#on} instead
```

Comment:
The `#` notation is Javadoc syntax and does not work in TypeDoc (even though it works in the IntelliJ editor), you need to use a `.` for static and instance accessors: `{@link PlayerAPI.on}`